code/modules/mob/living/simple_animal/hostile/cat_butcher.dm

```diff
@@ -33,90 +34,166 @@
+
+//cat butcher ai shit
+/mob/living/simple_animal/hostile/cat_butcherer/CanAttack(atom/the_target)
+	if(iscarbon(target))
+		var/mob/living/carbon/human/C = target
+		if(C.getorgan(/obj/item/organ/ears/cat) && C.getorgan(/obj/item/organ/tail/cat) && C.has_trauma_type(/datum/brain_trauma/severe/pacifism))//he wont attack his creations
+			if(C.stat && (!HAS_TRAIT(C, TRAIT_NOMETABOLISM) || !istype(C.dna.species, /datum/species/ipc)))//unless they need healing
+				return ..()
+			else
+				return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/cat_butcherer/MoveToTarget(list/possible_targets)
+	if(target)
+		if(ishuman(target))
+			var/mob/living/carbon/human/L = target
+			if(L.health <=30 || L.stat || !L.can_inject(null, FALSE)) // base health to move in to attack is 30, not 40, as it accounts for armor somewhat
+				retreat_distance = 0
+			else
+				retreat_distance = 3 //spam chems if they aren't low and can be injected
+		else 
+			retreat_distance = 0//dont shoot chems at mobs that dont give a fuck
+	return ..()
+
+/mob/living/simple_animal/hostile/cat_butcherer/PickTarget(list/Targets)
+	if(target != null)
+		for(var/pos_targ in Targets)
+			Targets[pos_targ] = 1
+			var/atom/A = pos_targ
+			var/target_dist = get_dist(targets_from, target)
+			var/possible_target_distance = get_dist(targets_from, A)
+			if(target_dist < max(possible_target_distance, 3))
+				Targets -= A
+	for(var/pos_targ in Targets)
+		Targets[pos_targ] = 1
+		if(ishuman(pos_targ))
+			var/mob/living/carbon/human/H = pos_targ
+			if(!CanAttack(H))
+				Targets -= H 
+				continue
+			if(H.stat == DEAD)
+				Targets -= H 
+				continue
+			if(H.stat)
+				Targets[H] = 20
+				continue
+			else
+				var/healthdiff = 10-round(H.health/10)
+				Targets[H] = CLAMP(healthdiff,1,12)
+	if(!Targets.len)//sanity check
+		return
+	var/chosen_target = pickweight(Targets)//Pick the remaining targets (if any) at random
+	return chosen_target
+
+/mob/living/simple_animal/hostile/cat_butcherer/death(gibbed)
+	if(LAZYLEN(victims) >= 5)
+		say("I made [LAZYLEN(victims)] creations! I have no regrets!!")
+	return..()
+//special darts below this point
+/obj/item/projectile/bullet/dart/tranq
+	name = "tranquilizer dart"
+
+/obj/item/projectile/bullet/dart/tranq/Initialize()
+	. = ..()
+	reagents.add_reagent(/datum/reagent/toxin/chloralhydrate, 4) //these'll get the victim wallslamming and then sleep em, but it will take awhile before it puts the victim to sleep
+
+/obj/item/projectile/bullet/dart/tranqplusplus
+	name = "tranquilizer dart"
+
+/obj/item/projectile/bullet/dart/tranqplus/Initialize()
+	. = ..()
+	reagents.add_reagent(/datum/reagent/toxin/chloralhydrate, 4)
+	reagents.add_reagent(/datum/reagent/pax, 1)
+
+/obj/item/projectile/bullet/dart/tranqplusplus
+	name = "tranquilizer dart"
+	
+/obj/item/projectile/bullet/dart/tranqplusplus/Initialize()
+	. = ..()
+	reagents.add_reagent(/datum/reagent/toxin/chloralhydrate, 4)
+	reagents.add_reagent(/datum/reagent/pax, 3)
```

Comment:
```suggestion
/obj/item/projectile/bullet/dart/tranq/plus

/obj/item/projectile/bullet/dart/tranq/plus/Initialize()
	. = ..()
	reagents.add_reagent(/datum/reagent/pax, 1)

/obj/item/projectile/bullet/dart/tranq/plusplus
	
/obj/item/projectile/bullet/dart/tranq/plusplus/Initialize()
	. = ..()
	reagents.add_reagent(/datum/reagent/pax, 3)
```

Come _on_. I didn't sign in GitHub to review things like this.

Reply:
you didnt need to, i considered doing this but i was just too lazy 🤷‍♀️

Reply:
> Come on. I didn't sign in GitHub to review things like this.

You're right